recipes/czmq/all/conanfile.py

```diff
@@ -129,18 +130,18 @@ def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "czmq")
         self.cpp_info.set_property("cmake_target_name", self._czmq_target)
         self.cpp_info.set_property("pkg_config_name", "libczmq")
-        prefix = "lib" if self._is_msvc and not self.options.shared else ""
-        self.cpp_info.libs = ["{}czmq".format(prefix)]
+        prefix = "lib" if is_msvc(self) and not self.options.shared else ""
+        self.cpp_info.libs = [f"{prefix}czmq"]
         if not self.options.shared:
             self.cpp_info.defines.append("CZMQ_STATIC")
         if self.settings.os in ["Linux", "FreeBSD"]:
             self.cpp_info.system_libs.extend(["pthread", "m"])
         elif self.settings.os == "Windows":
             self.cpp_info.system_libs.append("rpcrt4")
-        if not self.options.shared:
-            stdcpp_library = tools.stdcpp_library(self)
-            if stdcpp_library:
-                self.cpp_info.system_libs.append(stdcpp_library)
+        # if not self.options.shared:
+        #     stdcpp_library = stdcpp_library(self)
+        #     if stdcpp_library:
+        #         self.cpp_info.system_libs.append(stdcpp_library)
```

Comment:
Should this be removed?

Reply:
@jwillikers 
Oh..., I forget to remove these lines.
Thanks!